components/brave_wallet/common/web3_provider_utils.cc

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <vector>
+
+#include "base/values.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+#include "brave/components/brave_wallet/common/web3_provider_utils.h"
+
+namespace brave_wallet {
+
+absl::optional<mojom::EthereumChain> ParameterValueToEthereumChain(
```

Comment:
This is exactly duplicate of `ValueToEthereumChain`, please move  all `*ValueTo` and `ValueTo*` functions here and rename this file to be `value_conversion_utils.cc`

Reply:
This is not a complete copy, there are differences in the data structure, but I have found a way to make it look the same. Removed ParameterValueToEthereumChain and moved other fucntions to value_conversion_utils